src/services/MemberService.js

```diff
@@ -204,29 +204,35 @@ async function getProfileCompleteness (currentUser, handle, query) {
     }
     
   })
-
   // Push on the incomplete traits for picking a random toast to show
   if(!data.education){
     showToast.push("education")
   }
   if(!data.workHistory){
     showToast.push("workHistory")
   }
-  if(!data.bio){
-    showToast.push("bio")
-  }
   if(!data.gigAvailability){
     showToast.push("gigAvailability")
   }
 
-  if(member.verified){
+  // TODO: Do we use the short bio or the "description" field of the member object?
+  if(member.description) {
     completeItems += 1
-    data.verified=true
+    data.bio = true
   }
   else{
-    showToast.push("verified")
+    showToast.push("bio")
   }
 
+  // TODO: Turn this back on once verification is implemented
```

Comment:
Is this legacy?

Reply:
@kkartunov - For now, we aren't going to look at verification for the profile "completeness".  I'm assuming we will in the future, at which point we'll turn this back on.